dataprofiler/profilers/base_column_profilers.py

```diff
@@ -251,7 +251,9 @@ def report(self, remove_disabled_flag: bool = False) -> None:
         raise NotImplementedError()
 
 
-class BaseColumnPrimitiveTypeProfiler(with_metaclass(abc.ABCMeta, BaseColumnProfiler)):
+class BaseColumnPrimitiveTypeProfiler(
+    with_metaclass(abc.ABCMeta, BaseColumnProfiler)  # type: ignore
+):
     """Abstract class for profiling primative data type for col of data."""
 
     def __init__(self, name: Optional[str]) -> None:
```

Comment:
I'm not sure `name` is allowed to be optional. Reason for choosing Optional here?

Reply:
ahh do the tests do this?